### PR TITLE
Sefi Changes

### PR DESCRIPTION
**Pull request by Sefiraat**
Add DankTech2
Change DankTech to move to DankTech2
Change SMG links to the post RC27 fork

### CODE REVIEW

File: pages/Addons.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ DynaTech | ProfElements | <details><summary>expand</summary>An addon that adds r
...
-SimpleMaterialGenerators (SMG) | waleks647 | <details><summary>expand</summary>A simple addon with simple generators to reduce some grind.</details> | [Readme](https://github.com/waleks647/SMG/blob/master/README.md) | [GitHub](https://github.com/waleks647/SMG) | [Builds](https://thebusybiscuit.github.io/builds/waleks647/SMG/master)
+SimpleMaterialGenerators (SMG) | waleks647 | <details><summary>expand</summary>A simple addon with simple generators to reduce some grind.</details> | [Readme](https://github.com/Sefiraat/SMG/blob/master/README.md) | [GitHub](https://github.com/Sefiraat/SMG) | [Builds](https://thebusybiscuit.github.io/builds/Sefiraat/SMG/master)
```
Comment (svr333) [inline pages/Addons.md:59]:
```suggestion
SimpleMaterialGenerators (SMG) | waleks647, Sefiraat | <details><summary>expand</summary>A simple addon with simple generators to reduce some grind.</details> | [Readme](https://github.com/Sefiraat/SMG/blob/master/README.md) | [GitHub](https://github.com/Sefiraat/SMG) | [Builds](https://thebusybiscuit.github.io/builds/Sefiraat/SMG/master)
```

File: pages/Addons.md [hunk 2]
```diff
@@ -67,6 +67,7 @@ Galactifun | Seggan, Mooy1, GallowsDove, ProfElements | <details><summary>expand
...
 Crystamae Historia | Sefiraat | <details><summary>expand</summary>Crystamae Historia is a large-scale magic addon that adds a brand new series of crafting mechanics allowing players to craft magical gadgets, spells and tools. Crysta has a deep progression system, massive amounts of lore and enough spells/gadgets to keep any player engaged for a long, long, time.</details> | [Wiki](https://github.com/Sefiraat/CrystamaeHistoria/wiki) | [GitHub](https://github.com/Sefiraat/CrystamaeHistoria) | [Builds](https://thebusybiscuit.github.io/builds/Sefiraat/CrystamaeHistoria/master/)
+DankTech2 | Sefiraat | <details><summary>expand</summary>DankTech is a plugin that attempts to recreate the Dank Storage Mod and /Dank/Null/ items from Modded. Features include mass volume storage backpacks with auto-voiding, building, easy unloading and more!</details> | [Wiki](https://github.com/Sefiraat/DankTech2/wiki) | [GitHub](https://github.com/Sefiraat/DankTech2) | [Builds](https://thebusybiscuit.github.io/builds/Sefiraat/DankTech2/master/)
```
Comment (svr333) [inline pages/Addons.md:70]: There currently is no wiki so the wiki link points to the main github
not something that needs to be fixed just pointing it out

Comment (Sefiraat) [reply pages/Addons.md:70]: Yeah I need to get on the wiki at some point D:

File: pages/Addons.md (hunk 1 above)
Comment (Sefiraat) [reply pages/Addons.md:59]: Currently I don't feel like I've done enough to be worthy of having my name there, Waleks did all the work making it. I may add this in should we add further gens in - this change is focused on making sure new players download the correct version really

Comment (svr333) [reply pages/Addons.md:59]: sure but seeing it does link to your github I would just add it for consistency sake
but if you feel strongly about it then its fine too

File: pages/Addons.md (hunk 2 above)
Comment (svr333) [reply pages/Addons.md:70]: fair

**Review by svr333 — APPROVED** [review]

**Review by Phoenix-Starlight — APPROVED** [review]

**Pull request merged**